src/manager_impl/TaxiPoolManagerImpl.java

```diff
@@ -0,0 +1,38 @@
+package manager_impl;
```

Comment:
for implementation you need to use the same package but structure should be as
![image](https://user-images.githubusercontent.com/40631341/99153536-2f0bb100-26ba-11eb-9d09-3735fbb3fb44.png)